### ISSUE TITLE
(Difficult) Rewrite in language other than Python

### ISSUE THREAD

**Issue by jessiejs** [Darth-Ness/Katana-HTML-Renderer#19]
I find this pretty interesting, but it is written in python, and there is a couple downsides to that.

1. You need python installed to use it
2. Python is **really** slow, which is not what you want from a renderer.

Here are a few reccomendations I have

1. C#
2. C++
3. Java
4. Rust

I would be willing to help with the rewrite.

**Comment by Darth-Ness** [comment]
> I find this pretty interesting, but it is written in python, and there is a couple downsides to that.
> 
>     1. You need python installed to use it
> 
>     2. Python is **really** slow, which is not what you want from a renderer.
> 
> 
> Here are a few reccomendations I have
> 
>     1. C#
> 
>     2. C++
> 
>     3. Java
> 
>     4. Rust
> 
> 
> I would be willing to help with the rewrite.

Don't you need Windows for C# to work? For the .NET framework. Rust would work. However, I rarely work on this any more.

**Comment by CodeLikeCrazE** [comment]
No, C# is cross platform, it compiles to IL, the .NET intermediary language, which runs on all platforms (that's how Unity works on all platforms).

**Comment by jessiejs** [comment]
> > I find this pretty interesting, but it is written in python, and there is a couple downsides to that.
> > ```
> > 1. You need python installed to use it
> > 
> > 2. Python is **really** slow, which is not what you want from a renderer.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Here are a few reccomendations I have
> > ```
> > 1. C#
> > 
> > 2. C++
> > 
> > 3. Java
> > 
> > 4. Rust
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I would be willing to help with the rewrite.
> 
> Don't you need Windows for C# to work? For the .NET framework. Rust would work. However, I rarely work on this any more.

No, C# is cross platform, it compiles to IL, the .NET intermediary language, which runs on all platforms (that's how Unity works on all platforms).
